From 6acc63bff58c9a2a571f88ab61157c4cfef55410 Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Wed, 13 Nov 2024 14:35:36 -0500
Subject: [PATCH] kmod version fix

Signed-off-by: Scott Little <scott.little@windriver.com>
---
 Dockerfile.daemon | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Dockerfile.daemon b/Dockerfile.daemon
index facfa95..52ea883 100644
--- a/Dockerfile.daemon
+++ b/Dockerfile.daemon
@@ -26,7 +26,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o sriov_fe
 FROM registry.access.redhat.com/ubi9/ubi:9.4-947 AS package_installer
 
 RUN yum install \
-	kmod-28-9.el9 \
+	kmod-28-10.el9 \
 	pciutils-3.7.0-5.el9 \
 	procps-ng-3.3.17-14.el9 \
 	--releasever 9 --setopt install_weak_deps=false --nodocs -y && \
-- 
2.25.1

